app/controllers/registration/early_years_emails_controller.rb

```diff
@@ -6,12 +6,14 @@ def update
       form.early_years_emails = user_params[:early_years_emails]
 
       if form.save
+        track('user_early_years_emails_change', { success: true })
         if current_user.registration_complete?
           redirect_to user_path, notice: t(:details_updated)
         else
           complete_registration
         end
       else
+        track('user_early_years_emails', { success: false })
```

Comment:
The keys don't match

Reply:
[c335467](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1048/commits/c3354674814f7a1ae7b564ea8a69bff9ce76e82f)